scripts/histmakers/mw_with_mu_eta_pt.py

```diff
@@ -99,7 +101,7 @@
 axis_charge = common.axis_charge
 axis_passIso = common.axis_passIso
 axis_passMT = common.axis_passMT
-axis_mt = hist.axis.Variable([0,int(mtw_min/2.),mtw_min] + list(range(mtw_min+5, 95, 5)) + [100, 120], name = "mt", underflow=False, overflow=True)
+axis_mt = hist.axis.Variable((*np.arange(0, mtw_min, 2), *np.arange(mtw_min+5, 95, 5), 100, 120), name = "mt", underflow=False, overflow=True)
```

Comment:
I think the np.arange(0,40,2) gives you an array [0,2,4, ... 38] but we want [0,20,40]